app/components/DepositWithdraw/blocktrades/WithdrawModalBlocktrades.jsx

```diff
@@ -223,12 +223,23 @@ class WithdrawModalBlocktrades extends React.Component {
 
                 const {feeAmount} = this.state;
 
-                let amount = parseFloat(String.prototype.replace.call(this.state.withdraw_amount, /,/g, ""));
+                let amount = parseFloat(String.prototype.replace.call(this.state.withdraw_amount, /,/g, "")) + parseFloat(String.prototype.replace.call(this.props.gateFee, /,/g, ""));
+
                 let sendAmount = new Asset({
                     asset_id: asset.get("id"),
                     precision: asset.get("precision"),
                     real: amount
                 });
+                
+                let userAmount = new Asset({
+                    amount: this.props.balance.get("balance"),
+                    asset_id: this.props.asset.get("id"),
+                    precision: this.props.asset.get("precision")
+                });
+
+                console.log("Sending Amount: " + sendAmount.amount + " Available Amount: " + userAmount.amount);
+
+                sendAmount.amount = sendAmount.amount > userAmount.amount ? userAmount.amount : sendAmount.amount;
```

Comment:
If the user has less in their holdings than what they try to send, this line will set the sending about to what the user has. 

But I will look on the logic in `SimpleDepositWithdrawModal`, as you suggest, and supply a new PR.

Reply:
Oh right, I just looked at the console log when I tried and noticed the amount was more than my balalance.

Reply:
The console log should be removed though. Let me get an update for you later.

Reply:
I have submitted a new commit to some minor changes to the code on how we check if the user has a balance that is enough. 

Since checkBalance doesn't account for the gateFee value today, I would think a change to checkBalance would be the best way to go, but I don't want to change to much now, since it would be quite a change.

This would be a better way to go I think, since this will only apply to Withdrawls trough OL as of now. But since they are the only gateway that we have that take a fee, it is sufficient to rectify the issue at hand.

But we could revisit this later.

https://github.com/bitshares/bitshares-ui/blob/9691f28208eb4b39ff2e17ab2a40467261b9ca10/app/lib/common/trxHelper.js#L159